python-src/resources.py

```diff
@@ -1,6 +1,8 @@
 import datetime
 import random
 
+from google.appengine.api.channel import channel
```

Comment:
Did you manage to get it work? We can make a quick session tomorrow morning if you are free